Dockerfile

```diff
@@ -68,6 +96,9 @@ FROM final
 
 COPY src/ main/
 COPY Ctl/docker/entrypoint.sh .
+
+RUN . "$VIRTUAL_ENV"/bin/activate
```

Comment:
Don't need to activate because the PATH is already set.

Reply:
Oh I didn't realize that's how it worked, good to know!

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/7cb9e0cd7285023ca68f5d0f722153587a816fb3